src/components/Input/index.tsx

```diff
@@ -123,14 +161,22 @@ export const Input: React.FC<InputProps> = ({
         </label>
       )}
       <div className="mt-1 relative shadow-sm">
+        {type === 'money' && (
+          <div className="absolute inset-y-0 left-0 flex items-center pl-3 pointer-events-none">
+            <span className="text-gray-500 sm:text-sm">$</span>
+          </div>
+        )}
         <input
           name={label}
           id={label}
-          type={type === 'password' && passwordVisibility ? 'text' : type}
+          type={getType()}
           className={classNames(
             {
               'pr-16': type === 'password' && error,
               'pr-10': +(type === 'password') ^ +!!error,
+              'pl-7': type === 'money',
```

Comment:
use type === 'money' and type === 'password' too much times. maybe store it as isPassword and isMoney

Reply:
i feel like this wouldn't be such a good idea. assuming that you would need this for many different types. you'd have to store like 10 different booleans when in the code, it wouldn't become that much more readable.